src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.File.cs

```diff
@@ -38,26 +38,13 @@ public ProjectRootElement ProjectRoot
             }
         }
 
-        public string Sdk
-        {
-            get
-            {
-                var sdk = ProjectRoot.Sdk;
-
-                if (sdk is null)
-                {
-                    throw new ArgumentOutOfRangeException("Should check IsSdk property first");
-                }
-
-                return sdk;
-            }
-        }
-
         public bool IsSdk =>
```

Comment:
I don't know if this is necessary anymore. Shouldn't it be equivalent to `Sdk.Any()`?

Reply:
Changed to sdk.Any().

Reply:
Can we remove it? A caller could call `Sdk.Any()`

Reply:
Removed